tests/unit/test_extras.hxx

```diff
@@ -153,6 +153,7 @@ public:
     maxregionblocksize = MAXREGIONBLOCKSIZE;
 
     setCoordinates(nullptr);
+    setParallelTransform(bout::utils::make_unique<ParallelTransformIdentity>(*this));
```

Comment:
This is undefined behaviour, btw, as `this` is not a complete `Mesh` object until _after_ the constructor has finished. If we do need to do this, it should be done in `FakeMeshFixture`